about/list_of_features.rst

```diff
@@ -22,7 +22,7 @@ Platforms
 
 **Can run both the editor and exported projects:**
 
-- Windows (x86, 64-bit and 32-bit).
+- Windows (x86 and ARM, 64-bit and 32-bit).
```

Comment:
I don't think Godot supports 32-bit Arm Windows. It may be better to list the architectures explicitly.

```
- Windows (x86_32, x86_64, arm64).
- macOS (x86_64 and arm64).
- Linux (x86_32, x86_64, arm32, arm64).
```